docs/manpage_syntax.md

```diff
@@ -0,0 +1,102 @@
+% podman-command(1)
+
+## NAME
+podman\-command - short description
+
+## SYNOPSIS
+(Shows the command structure.)
+
+**podman command** [_optional_] _mandatory value_
```

Comment:
Just checking, so these really should be `_optional_` and not `*optional*`?  We have `*optional*` all over the place.

Reply:
or maybe   `**_optional_**`

Reply:
Yeah I believe you're right. Markdown applications don’t agree on how to handle underscores in the middle of a word. So in this specific case it wouldn't matter, but the the general idea I would change it to your suggestion.
But i wouldn't make it bold, since the command itself is bold.